core/src/test/java/com/linecorp/armeria/client/proxy/ProxyClientIntegrationTest.java

```diff
@@ -331,12 +336,32 @@ void testHttpProxyPrefaceFailure() throws Exception {
         clientFactory.close();
     }
 
-    @Test
-    void testHttpsProxyBasicCase() throws Exception {
+    @ParameterizedTest
+    @EnumSource(value = SessionProtocol.class, mode = Mode.INCLUDE, names = {"H1C", "H2C", "HTTP"})
+    void testHttpsProxyBasicCase(SessionProtocol protocol) throws Exception {
```

Comment:
How about `testHttpToHttpsEndpoint`? so that people get easily noticed the differences with the test below.
Or I think we can just combine these two tests into one.